### PR TITLE
cemu-ti: 1.3 -> unstable-2022-06-29

### LINKED ISSUE #188193: nixpkgs#cemu fails building
### Describe the bug
See title

### Steps To Reproduce
```nix
{
  inputs.nixpkgs.url = "github:nixpkgs/nixos/b9fd420fa535fd254c6f1f26df770b32a9dc98b4";

  outputs =
    { self, nixpkgs }:
    {
      packages.x86_64-linux.default = nixpkgs.legacyPackages.x86_64-linux.cemu;
    };
}
```

Fails compilation on x86_64-linux with
```
CEmu> /nix/store/zgvxnf9047rdd8g8kq2zxxm9k6kfqf8b-binutils-2.38/bin/ld: /nix/store/rqfa4jigjlb3s59g9i5554sjqdr53w8z-qtbase-5.15.3/lib/libQt5Core.so: undefined reference to `std::__exception_ptr::exception_ptr::_M_release()@CXXABI_1.3.13'
CEmu> /nix/store/zgvxnf9047rdd8g8kq2zxxm9k6kfqf8b-binutils-2.38/bin/ld: /nix/store/rqfa4jigjlb3s59g9i5554sjqdr53w8z-qtbase-5.15.3/lib/libQt5Widgets.so: undefined reference to `std::__throw_bad_array_new_length()@GLIBCXX_3.4.29'
CEmu> /nix/store/zgvxnf9047rdd8g8kq2zxxm9k6kfqf8b-binutils-2.38/bin/ld: /nix/store/rqfa4jigjlb3s59g9i5554sjqdr53w8z-qtbase-5.15.3/lib/libQt5Core.so: undefined reference to `std::__exception_ptr::exception_ptr::_M_addref()@CXXABI_1.3.13'
CEmu> collect2: error: ld returned 1 exit status
CEmu> make: *** [Makefile:507: CEmu] Error 
```

### Expected behavior
It should build. Or it should be marked as broken.

### Notify maintainers
@luc65r 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 5.18.16, NixOS, 22.11 (Raccoon), 22.11.20220816.762b003`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.10.3`
 - nixpkgs: `/run/current-system/sw/nixpkgs`
```

### PR DESCRIPTION
**Pull request by luc65r**
###### Description of changes

Renamed from `cemu` to `cemu-ti`.

Fixes https://github.com/NixOS/nixpkgs/issues/188193.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/science/math/cemu-ti/default.nix [hunk 1]
```diff
@@ -1,52 +1,40 @@
...
   qmakeFlags = [
     "gui/qt"
-    "CONFIG+=ltcg"
   ];
 
   meta = with lib; {
-    changelog = "https://github.com/CE-Programming/CEmu/releases/tag/v${version}";
     description = "Third-party TI-84 Plus CE / TI-83 Premium CE emulator, focused on developer features";
     homepage = "https://ce-programming.github.io/CEmu";
     license = licenses.gpl3;
```
Comment (azahi) [inline pkgs/applications/science/math/cemu-ti/default.nix:40]: While we are at it: https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206

**Review by azahi — CHANGES_REQUESTED** [review]

File: pkgs/applications/science/math/cemu-ti/default.nix (hunk 1 above)
Comment (thiagokokada) [inline pkgs/applications/science/math/cemu-ti/default.nix:40]:
```suggestion
    license = licenses.gpl3Plus;
```

**Comment by thiagokokada** [comment]
Result of `nixpkgs-review pr 188939` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cemu-ti</li>
  </ul>
</details>

**Review by thiagokokada — APPROVED** [review]

**Comment by thiagokokada** [comment]
Don't know how to actually test the program, but I ran here and it seems to works fine.

**Comment by winterqt** [comment]
We should probably include something in the changelog about this rename/the emulator replacing it, even if nobody has depended on it in months. cc @mweinelt, thoughts?

**Comment by thiagokokada** [comment]
> We should probably include something in the changelog about this rename/the emulator replacing it, even if nobody has depended on it in months. cc @mweinelt, thoughts?

Makes sense, maybe it is better to do this in #198590 though.

**Comment by SuperSamus** [comment]
Should #114509 also be closed?

**Pull request merged**